_posts/success_stories/2024-05-20-lavinmq-understanding-code.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "LavinMQ: Understanding code to get the best performance"
+summary: "Building LavinMQ in Crystal was an active choice, as maintaining control over the codebase was a high priority."

```

Comment:
**thought:** To me this does not feels very self-concious, it has a bit a tune of an apology maybe.

How about something more catchy?
```suggestion
summary: "Crystal was chosen for maintaining control over the entire codebase."
```